internal/printer/printer_test.go

```diff
@@ -1093,6 +1160,19 @@ import * as $$module1 from 'react-bootstrap';`},
 			if len(tt.want.frontmatter) > 0 {
 				toMatch += test_utils.Dedent(tt.want.frontmatter[0])
 			}
+			// Fixes some tests where getStaticPaths appears in a different location
+			if tt.want.skipHoist == true && len(tt.want.getStaticPaths) > 0 {
+				toMatch += "\n\n"
+				toMatch += strings.TrimSpace(test_utils.Dedent(tt.want.getStaticPaths)) + "\n"
```

Comment:
Fortunately the `skipHoist` “hack“ isn’t too bad. It only duplicates the `getStaticPaths` printing, which is workable.

The other additions here handle the cleanup of having to declare modules twice.